packages/blade/src/components/ProgressBar/CircularProgressBar.web.tsx

```diff
@@ -19,13 +19,13 @@ import getBaseTextStyles from '~components/Typography/BaseText/getBaseTextStyles
 
 const pulseKeyframes = keyframes`
   0% {
-    opacity: 100;
```

Comment:
wow how did this even happen?

Reply:
I just saw this & fixed it. This wasn't causing any issues though.